modules/admin/ref-rbac-conditional-policy-definition.adoc

```diff
@@ -0,0 +1,347 @@
+            "description": "Allow entities owned by a specified claim",
+            "resourceType": "catalog-entity",
+            "paramsSchema": {
+               "type": "object",
+               "properties": {
+                  "claims": {
+                     "type": "array",
+                     "items": {
+                        "type": "string"
+                     },
+                     "description": "List of claims to match at least one on within ownedBy"
+                  }
+               },
+               "required": [
+                  "claims"
+               ],
+               "additionalProperties": false,
+               "$schema": "http://json-schema.org/draft-07/schema#"
+            }
+         }
+      ]
+   }
+   ... <another plugin condition parameter schemas>
+]
+----
+
+The RBAC backend API constructs a condition JSON object based on the previous condition schema.
+
+== Examples of conditional policies
+
+In {product}, you can define conditional policies with or without criteria, such as:
+
+A condition without criteria::
++
+--
+Consider a condition without criteria displaying catalogs only if user is a member of the owner group. To add this condition, you can use the catalog plugin schema `IS_ENTITY_OWNER` as follows:
+
+.Example condition without criteria
+[source,json]
+----
+{
+  "rule": "IS_ENTITY_OWNER",
+  "resourceType": "catalog-entity",
+  "params": {
+    "claims": ["group:default/team-a"]
+  }
+}
+----
+
+In the previous example, only one conditional parameter is used: `claims`, which contains a list of user or group entity references.
+
+You can apply the previous example condition to the RBAC REST API by adding additional parameters as follows:
+
+[source,json]
+----
+{
+  "result": "CONDITIONAL",
+  "roleEntityRef": "role:default/test",
+  "pluginId": "catalog",
+  "resourceType": "catalog-entity",
+  "permissionMapping": ["read"],
+  "conditions": {
+    "rule": "IS_ENTITY_OWNER",
+    "resourceType": "catalog-entity",
+    "params": {
+      "claims": ["group:default/team-a"]
+    }
+  }
+}
+----
+--
+
+A condition with criteria::
++
+--
+Consider a condition with criteria, which displays catalogs only if user is a member of owner group OR displays list of all catalog user groups.
+
+To add the criteria, you can add another rule as `IS_ENTITY_KIND` in the condition as follows:
+
+.Example condition with criteria
+[source,json]
+----
+{
+  "anyOf": [ 
+    {
+      "rule": "IS_ENTITY_OWNER",
+      "resourceType": "catalog-entity",
+      "params": {
+        "claims": ["group:default/team-a"]
+      }
+    },
+    {
+      "rule": "IS_ENTITY_KIND",
+      "resourceType": "catalog-entity",
+      "params": {
+        "kinds": ["Group"]
+      }
+    }
+  ]
+}
+----
+
+[NOTE]
+====
+Running conditions in parallel during creation is not supported. Therefore, consider defining separate condition requests or nesting your conditions based on the available criteria.
```

Comment:
```suggestion
Running conditions in parallel during creation is not supported. Therefore, consider defining nested conditional policies based on the available criteria.
```

I ended up getting some clarification from @AndrienkoAleksandr and it seems like we do not support separate conditional requests. So, I need to update the documentation in our plugin as well.

Reply:
Done.